### PR TITLE
Add zpages and pprof data documentation

### PR DESCRIPTION
**Pull request by mackjmr**
### What does this PR do?
The data was added to the flare in #28848 and #28927. This documents it.

### CODE REVIEW

File: comp/otelcol/ddflareextension/README.md [hunk 1]
```diff
@@ -43,14 +43,22 @@ The flare will collect all environment variables, and these can be found in `ote
...
 
 ```
 {"status":"Server available","upSince":"2024-08-14T14:54:00.575804+02:00","uptime":"28.470434291s"}
 ```
 - pprof: Found in `otel/otel-flare/pprof`
+
+Contains a allocs (`dd-autoconfigured_debug_pprof_allocs`), heap (`dd-autoconfigured_debug_pprof_heap`) and cpu (`dd-autoconfigured_debug_pprof_profile`) profile. Profiles can be opened with the pprof tool, e.g.
+```
+go tool pprof -http=: otel/otel-flare/pprof/dd-autoconfigured_debug_pprof_heap.dat
+```
```
Comment (dineshg13) [inline comp/otelcol/ddflareextension/README.md:56]: can u add local host link or link to pprof documentation

**Review by dineshg13 — APPROVED** [review]
Approved with a  comment

File: comp/otelcol/ddflareextension/README.md (hunk 1 above)
Comment (mackjmr) [reply comp/otelcol/ddflareextension/README.md]: Added link 👍

**Pull request merged**